example/scripts/generate_ingest/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@cumulus/example/generate_ingest",
+  "version": "4.0.0",
+  "description": "Script to generate test data for scaled ingest",
+  "keywords": [
+    "GIBS",
+    "CUMULUS",
+    "NASA"
+  ],
+  "types": "./index.d.ts",
+  "engines": {
+    "node": ">=12.18.0"
+  },
+  "publishConfig": {
+    "access": "private"
+  },
+  "homepage": "https://github.com/nasa/cumulus/tree/master/packages/types#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nasa/cumulus",
+    "directory": "packages/types"
+  },
+  "devDpendencies": {
+    "@cumulus/aws-client": "4.0.0",
+    "@cumulus/common": "4.0.0"
```

Comment:
Are these really devDeps?

Reply:
Probably not.  Functionally it doesn't matter as it's only being run in a dev env.  Still. 
0661434ca